lib/utils/thumbnail_util.dart

```diff
@@ -42,6 +42,18 @@ Future<Uint8List?> getThumbnail(EnteFile file) async {
   }
 }
 
+Future<File?> getThumbnailFile(EnteFile file) async {
+  final thumbnail = await getThumbnail(file);
+  if (thumbnail != null) {
+    final cachedThumbnail = cachedThumbnailPath(file);
+    if (!await cachedThumbnail.exists()) {
+      await cachedThumbnail.writeAsBytes(thumbnail, flush: true);
+    }
+    return cachedThumbnail;
+  }
```

Comment:
`cachedThumbnailPath` will throw error if this is called for a file that's not uploaded.
May be rename this to `getRemoteThubnailFile()`?